paddle/fluid/operators/detail/mpi_client.cpp

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
Separate the implement and design to 2 PRs.

Reply:
I have deleted the implement now and will rebuild it in another PR.